esi_leap/api/controllers/v1/node.py

```diff
@@ -66,33 +66,41 @@ def get_all(self):
 
         project_list = keystone.get_project_list()
         now = datetime.now()
+
+        offers = offer_obj.Offer.get_all({"status": "available"},
+                                         context)
+
+        f_leases = lease_obj.Lease.get_all({"status": [statuses.CREATED]},
+                                           context)
+
+        f_lease_uuids = ' '
```

Comment:
I don't think you need this

Reply:
`File "/home/rohand/esi-leap/.tox/py3/lib/python3.10/site-packages/mock/mock.py", line 1346, in patched
    return func(*newargs, **newkeywargs)

      File "/home/rohand/esi-leap/esi_leap/tests/api/controllers/v1/test_node.py", line 50, in test_get_all
    data = self.get_json("/nodes")

      File "/home/rohand/esi-leap/esi_leap/tests/api/base.py", line 77, in get_json
    response = self.app.get(full_path,

      File "/home/rohand/esi-leap/.tox/py3/lib/python3.10/site-packages/webtest/app.py", line 324, in get
    return self.do_request(req, status=status,

      File "/home/rohand/esi-leap/.tox/py3/lib/python3.10/site-packages/webtest/app.py", line 646, in do_request
    self._check_status(status, res)

      File "/home/rohand/esi-leap/.tox/py3/lib/python3.10/site-packages/webtest/app.py", line 675, in _check_status
    raise AppError(

    webtest.app.AppError: Bad response: 400 Bad Request (not 200 OK or 3xx redirect for http://localhost/v1/nodes)
b'{"faultcode": "Client", "faultstring": "Invalid input for field/attribute future_leases. Value: \'[]\'. Wrong type. Expected \'<class \'str\'>\', got \'<class \'list\'>\'", "debuginfo": null}'

`
From stack trace I think it is on line 47 in tests/api/controllers/v1/test_node.py

Reply:
Okay! I'll take a look when you submit the updated PR.

Reply:
Oh, you already submitted the updated PR. I'll take a look now!